index.html

```diff
@@ -4,8 +4,82 @@
      <meta charset="UTF-8">
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      <title>project 3</title>
+     <link rel="stylesheet" href="style.css">
+   <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Kanit:wght@300&display=swap" rel="stylesheet">
  </head>
  <body>
-     
+     <header>
+       <h1>
+        Design conference<br>
+         ——Typography In Web Design
+       </h1>
+      </header>
+       <nav>
+      <ul class="navigation" >  
+      <li><a href="#">Home</a></li>
+      <li><a href="#">Speakers</a></li>
+      <li><a href="#">Register</a></li>
+      <li><a href="#">Contact</a></li>
+      </ul> 
+      </nav>
+   <main>
+     <ul class="u-list-reset ">
+    <li class="talk1">
+     <img src="https://cdn.glitch.com/0e85a349-5917-49b5-9721-e8ec891917e8%2F1.png?v=1607879901347"
+        alt="Photo of Jordan Devos" />
+     <h3>Jordan Devos</h3>
+     <h4>Talk: Designing for Readability – A Guide to Web Typography<br>
+       Time：10 AM - 12 PM, February 1st, 2021
+     </h4>
+     <h6>
+       About the speaker: Jordan is an expert designer whose focus is on visual communications and whose work integrates strategy, service design, and UX/UI. She has worked in a range of companies, from corporate in-house to boutique design studios. A self-starter who wears many hats, Jordan is in her element leading a project or managing a team.
+     </h6>
+     <p>Description: Traditionally, graphic designers have always studied the fundamentals of typography in terms of print design. But as the role of the designer now spans across all types of digital mediums, typography is a bigger beast to tame. In this talk, Devos will introduce some web typography tips about the web typeface and formatting which leads to the best reading experience.</p>
+      </li>
+    
+      <li class="talk2"> 
+     <img src="https://cdn.glitch.com/0e85a349-5917-49b5-9721-e8ec891917e8%2FPhoto%20of%20Suzanne%20Scacca.png?v=1607880365444"
+        alt="Photo of Suzanne Scacca" />
+      <h3>Suzanne Scacca</h3>
+     <h4>A Reference Guide For Typography In Mobile Web Design <br>
+       Time：13 PM - 15PM, February 1st, 2021
+     </h4>
+     <h6>About the speaker: Suzanne Scacca is a former WordPress implementer, trainer and agency manager who now works as a freelance copywriter. She specializes in crafting marketing, web design, WordPress and millennial-related content for a variety of publications.</h6>
+     <p>Description: In terms of how to handle typography in mobile web design, it appears that simpler and safer works best. In this talk, Scacca will lead to break down the elements designers need to pay attention to in mobile typography and then visit what the research says about how to handle them. </p>
+       </li>
```

Comment:
Since the markup for each talk is similar, everything I wrote above is also true for the following talks, too.